### PR TITLE
DownloadPackage to support latest version 

### LINKED ISSUE #12333: DownloadPackage doesn't support latest version
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Feature

**Enter Task Name**: DownloadPackage

- Server - Azure Pipelines or TFS on-premises?
Azure DevOps Server 2019.1.1

## Issue Description

DownloadPackage doesn't support selecting the latest package at runtime

### PR DESCRIPTION
**Pull request by evertonmc**
Fixes #12333 
I've added a 'latest version resolver' to select the current package at runtime.

- resolve if input version is `latest`
- Support latest from view
- added info at helpMarkDown

### CODE REVIEW

**Comment by joaquin386** [comment]
Very useful new feature. Right now have to do some hacking around to get it working with a 'latest' like behaviour.

**Comment by evertonmc** [comment]
gently pinging reviewers @aasim @phil-hodgson @shubham90 @satbai @20shivangi

**Comment by VOVELEE** [comment]
Hi guys,

do you expect this functionality to be reviewed soon? It will greatly help for a use-case I am developing.

**Comment by gogbg** [comment]
This is a very important functionality for my use case. Please make it so.

**Comment by VOVELEE** [comment]
gently pinging reviewers @aasim @phil-hodgson @shubham90 @satbai @20shivangi

**Comment by ChrisGibson1982** [comment]
Hi Guys, has these been any progress on this? @aasim @phil-hodgson @shubham90 @satbai @20shivangi

File: Tasks/DownloadPackageV1/task.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "author": "ms-vscs-rm",
     "version": {
         "Major": 1,
-        "Minor": 165,
```
Comment (satbai) [inline Tasks/DownloadPackageV1/task.json:12]: please update to Minor 174 Patch 0

**Comment by khdebje** [comment]
Can someone please resolve the remaining conflicts so the merge can be completed! This feature is fundamental for situations where npm is used to deploy custom build tools that are maintained. Not having to manually update ALL pipelines that are dependent on the latest version of a package would make my day so much nicer!

File: Tasks/DownloadPackageV1/task.json (hunk 1 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/task.json:12]: Please update it now to 176 and commit before the end of the sprint

**Review by aasim — CHANGES_REQUESTED** [review]
@evertonmc can you update the task version please?

File: Tasks/DownloadPackageV1/task.json (hunk 1 above)
Comment (evertonmc) [reply Tasks/DownloadPackageV1/task.json:12]: done

**Comment by evertonmc** [comment]
@aasim Done

**Review by aasim — APPROVED** [review]

**Comment by khdebje** [comment]
Cool stuff, thanks for the quick resolution! I'm still new to this, when can I expect the change to take affect?

**Comment by VOVELEE** [comment]
When can we expect this change to be released?

**Comment by satbai** [comment]
It's impossible to give an exact date, but as a rough estimate, you should start seeing the change within the first weeks of October.

**Comment by jfichtner** [comment]
I assume this should be active now? I'm getting an error when I set version to "latest" on a universal package feed.

**Comment by khdebje** [comment]
It works on my pipeline!

**Pull request merged**